### PR TITLE
toolbar separator overlapping with other items

### PR DESCRIPTION
**Pull request by alanrenmsft**
Fixes https://github.com/microsoft/azuredatastudio/issues/15981

before 
image

after:
image

the offending change was introduced in this PR: https://github.com/microsoft/azuredatastudio/pull/10271, I am not sure why is it needed, I don't see any other impact to the notebook UI after removing it.

### CODE REVIEW

**Comment by chlafreniere** [comment]
@alanrenmsft this just makes the separator smaller and shifts the other actions on the actionbar to the right, correct?

**Comment by alanrenmsft** [comment]
@chlafreniere yes, after removing this piece of css, the default margin will be applied, 6px for top and bottom, 8px for left and right.

**Comment by alanrenmsft** [comment]
image

@chlafreniere

**Review by Charles-Gagnon — APPROVED** [review]

File: src/sql/workbench/contrib/notebook/browser/notebook.css [hunk 1]
```diff
@@ -7,10 +7,6 @@
 	padding-top: 16px;
 }
 
-.notebookEditor .taskbarSeparator {
```
Comment (chlafreniere) [inline src/sql/workbench/contrib/notebook/browser/notebook.css:10]: Can we keep the separator height as it was to match the original designs?

Comment (alanrenmsft) [reply src/sql/workbench/contrib/notebook/browser/notebook.css:10]: I think all the toolbars in ADS should have the same style, personally I prefer to have some margin, for example:
image

@radentonev , what do you think?

**Comment by alanrenmsft** [comment]
for now, keep the size of the separator for notebook same as before until we hear back from Raden.

**Review by chlafreniere — APPROVED** [review]

**Comment by radentonev** [comment]
@alanrenmsft From looking at the screenshots, it looks like you are making the run all focus area taller and wider. You are also adding some extra spacing between that and the separator. Are these the only 2 discussed changes? 

In our design, we have the space between the "Run a" label and the separator to be 16px. I don't think we should change this, as it is coming from the wider ADUX guidelines and we are align on it with our partners. 

Screen Shot 2021-07-07 at 6 09 26 PM

**Comment by alanrenmsft** [comment]
@radentonev I just updated the screenshot using the latest insider. my original question is whether the separator should take up all the available vertical space(in the screenshot) or it should have some margin like the other places, I think this should be consistent across the ADS. For example:
image

Also in general, what should be the space between separator and the buttons?

**Comment by radentonev** [comment]
@alanrenmsft in terms of vertical height of the separator, I think it should be kept the same as in the screenshot. Reasons for this:

1. The separator component comes from Fluent, so many other teams are using the same one
2. The taller separator helps make it be more distinguishable from the other elements and thus serves its roles of splitting up the content better

As far as horizontal spacing goes, we have 16px as the space between separator and the elements before and after it.

You are right that we need to be consistent. The Query editor has an outdated design, as when we can scope this in, we should update it to match the notebook look & feel.

**Pull request merged**